massa-event-cache/src/config.rs

```diff
@@ -0,0 +1,14 @@
+use std::path::PathBuf;
+
+pub struct EventCacheConfig {
+    /// Path to the hard drive cache storage
+    pub event_cache_path: PathBuf,
+    /// Maximum number of entries we want to keep in the event cache
+    pub event_cache_size: usize,
```

Comment:
maybe a more explicit name (size makes me think of a number of bytes for caching): for example max_event_cache_length

Reply:
renamed